constants/index.ts

```diff
@@ -16,5 +16,6 @@ export const RESPONSE_MESSAGES = {
   NAME_ALREADY_EXISTS_400: { statusCode: 400, message: 'Button name already exists for this user.' },
   ADDRESSES_NOT_PROVIDED_400: { statusCode: 400, message: "'addresses' not provided." },
   INVALID_CHAIN_SLUG_400: { statusCode: 400, message: 'Invalid chain slug.' },
-  INVALID_BUTTON_DATA_400: { statusCode: 400, message: "'buttonData' is not valid JSON." }
+  INVALID_BUTTON_DATA_400: { statusCode: 400, message: "'buttonData' is not valid JSON." },
+  ADDRESS_NOT_PROVIDED_400: { statusCode: 400, message: 'Missing Address parameter' }
```

Comment:
To keep consistency with the rest of the `..._NOT_PROVIDED_400` errors, I think this error message would be better as `"'address' not provided"`.